README.md

```diff
@@ -42,15 +42,21 @@ The intended usage is as an npm script:
 }
 ```
 
+Then run it with: `$ npm run -s eslint-find-option-rules` (the `-s` is to silence npm output).
+
 ```
 available options are -c|--current, -a|--all-available, -p|--plugin, -u|--unused
-available flag is -n|--no-error
+available flags are -n|--no-error, --no-core, and -i/--include deprecated
 ```
 
 By default it will error out only for `-u|--unused`,
-however if you do not want the `process` to `exit` with a `non-zero` exit code, use the `-n|--no-error` along with `-u|--unused`
+however if you do not want the `process` to `exit` with a `non-zero` exit code, use the `-n|--no-error` flag along with `-u|--unused`.
+
+By default, core rules will be included in the output of `-c|--current`, `-a|--all-available`, and `-u|--unused`.  If you want to report on plugin rules only, use the `--no-core` flag.
+
+By default, deprecated rules will be omitted from the output of `-a|--all-available`, `-p|--plugin` and `-u|--unused`.  If you want to report on deprecated rules as well, use the `--include deprecated` or `-i deprecated` flag.
```

Comment:
yargs works both ways.  I updated the test and the doc to use the `=`, if only to have Travis try again and hopefully pass this time.